test/routes/review.test.js

```diff
@@ -238,6 +238,69 @@ describe('/review/', () => {
           expect(res.text).toContain('/hdc/send/finalChecks/1')
         })
     })
+    test('to contain version 1 of the main occupier question', () => {
+      licenceService.getLicence = jest.fn().mockReturnValue({
+        licence: {
+          proposedAddress: {
+            curfewAddress: {
+              addresses: [{ consent: 'Yes' }],
+            },
+          },
+          curfew: {
+            curfewAddressReview: {
+              version: '1',
+            },
+          },
+        },
+        stage: 'PROCESSING_RO',
+      })
+      conditionsService.populateLicenceWithConditions = jest.fn().mockReturnValue({
+        curfew: {
+          curfewAddressReview: {
+            version: '1',
+            addressReviewComments: '',
+          },
+        },
+      })
+
+      return request(app)
+        .get('/hdc/review/licenceDetails/1')
+        .expect(200)
+        .expect('Content-Type', /html/)
+        .expect((res) => {
+          expect(res.text).toContain('Does the main occupier consent to HDC?')
+        })
+    })
+
+    test('to contain version 2 of the main occupier question', () => {
```

Comment:
Think we should be consistent when referring to the content - see `he main occupier question` and `occupiers consent to HDC text` or are they referring to different things? Maybe wording could be a little clearer.

Reply:
agreed. I'll re do them